tools/fog-local-network/README.md

```diff
@@ -33,7 +33,7 @@ In order to use it, the following steps are necessary.
 
 3) Bootstrap a ledger:
     ```
-    cargo run --manifest-path ../util/generate-sample-ledger/Cargo.toml --release -- --txs 100
+    cargo run mc-util-generate-sample-ledger --release -- --txs 100
```

Comment:
Running this command as-is fails with this error: `cargo run could not determine which binary to run`.

If you change it to `cargo run -p ...` then it works.

Reply:
oh yeah that's a mistake thank you